src/strongbox/curseforge_api.clj

```diff
@@ -51,6 +51,13 @@
         ]
     (mapv rename-release release-list)))
 
+(defn game-version-flavor-to-game-track
```

Comment:
defn-spec
tests

should the default value be `:retail`? what happens when the given `game-version-flavor` isn't known?